src/ui/public/field_editor/field_editor.js

```diff
@@ -449,9 +450,18 @@ export class FieldEditorComponent extends PureComponent {
         >
           <p>
             <FormattedMessage
-              id="common.ui.fieldEditor.deleteField.label"
-              defaultMessage="You can't recover a deleted field.{newLine}Are you sure you want to do this?"
-              values={{ newLine: <p/> }}
+              id="common.ui.fieldEditor.deleteField.label.detail"
+              defaultMessage="You can't recover a deleted field. {question}"
+              values={{
+                question: (
+                  <p>
```

Comment:
Can we split this into a sibling paragraph? It's invalid HTML to have paragraphs inside of other paragraphs.